src/utils/share.js

```diff
@@ -0,0 +1,17 @@
+export function share(opts) {
```

Comment:
`export default` perhaps?

Reply:
Eh I figured there might be other share utils at some point